### PR TITLE
Add favorite test and fix aria-label

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add favorite test and fix aria-label

Co-authored by AJ

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

`npm run cypress:open`

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-add-favorite-test

File: packages/probers/cypress/e2e/favoriteTrack.cy.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import track from '../fixtures/track.json'
+import user from '../fixtures/user.json'
+
+function waitForTransaction(transactionNumber: number) {
```
Comment (dylanjeffers) [inline packages/probers/cypress/e2e/favoriteTrack.cy.ts:4]: Sweet! is there a way to share this?

**Review by dylanjeffers — APPROVED** [review]
nice job you two

File: packages/probers/cypress/e2e/favoriteTrack.cy.ts (hunk 1 above)
Comment (raymondjacobson) [reply packages/probers/cypress/e2e/favoriteTrack.cy.ts:4]: 👁️ 👄 👁️ 
if u must

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-add-favorite-test

**Pull request merged**